auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -134,14 +149,27 @@ export default defineComponent({
     }
 
     /**
-     * Save button is clicked. Update the dissolution batch size.
+     * Save button is clicked. Update the dissolution batch size job.
      * Only save if the inputted number is valid.
-     * TODO: Implement logic (job) once the BE is done.
+     * Show spinner and block buttons.
      */
-    const saveBtnClicked = (): void => {
+    const saveBtnClicked = async (): Promise<void> => {
       if (state.numberOfBusinessesRef.validate()) {
-        // staffStore.updateDissolutionBatchSize(state.numberOfBusinesses)
-        state.numberOfBusinessesNonEdit = state.numberOfBusinessesEdit
+        state.isSaving = true  // show the spinner
+        // Update store configurations array with the new number of inputted batch size
+        staffStore.involuntaryDissolutionConfigurations.configurations.find((config, index) => {
```

Comment:
This is cleanest way I could think of. Basically finding where the `NUM_DISSOLUTIONS_ALLOWED` is and changing it's value.